light_curves/code_src/helpers/scale_up.py

```diff
@@ -201,8 +293,30 @@ def _build_other(keyword, **kwargs_dict):
 
 
 def _construct_kwargs_dict(**kwargs_dict):
-    """Construct a complete kwargs dict by combining defaults, yaml (if requested), and `kwargs_dict`
-    (listed in order of increasing precedence).
+    """
+    Construct a complete kwargs dictionary by combining default values, YAML configuration (if requested), 
+    and provided keyword arguments, with precedence in that order.
+
+    Parameters
+    ----------
+    kwargs_dict : dict
+        A dictionary of keyword arguments provided by the user. These values take the highest 
+        precedence and will overwrite defaults and YAML configurations.
+
+    Returns
+    -------
+    dict
+        A dictionary containing the combined and final keyword arguments for the run, sorted by key.
+
+    Notes
+    -----
+    Default values are defined in the `DEFAULTS` dictionary.
+    If `use_yaml` is True in `kwargs_dict` or in the defaults, the function will load additional keyword 
```

Comment:
Same, all of these should be double backticks. But 1) we don't build docs from these, 2) these are private functions so even if we made API docs, these would not be included. 3) this really is a numpydoc bug/known issue.